pipelines/managed/push-disk-images-to-cdn/README.md

```diff
@@ -20,10 +20,17 @@ Tekton Pipeline to push disk images to a cdn using pulp
 | taskGitUrl                      | The url to the git repo where the release-service-catalog tasks to be used are stored                  | Yes | https://github.com/konflux-ci/release-service-catalog.git |
 | taskGitRevision                 | The revision in the taskGitUrl repo to be used                                                         | No       | -               |
 
+## Changes in 1.6.0
+* Update all tasks that now support trusted artifacts to specify the stepAction* parameters
+
 ## Changes in 1.5.0
 * Update the pipeline to add taskGiturl and taskGitRevison parameters to
   push-disk-images task.
 
+## Changes in 1.5.0
```

Comment:
there is already a 1.5.0 in the changelog

Reply:
fixed